tests/status_response/__init__.py

```diff
@@ -16,7 +16,7 @@ class BaseStatusResponseTest(abc.ABC):
     # if we don't specify item in raw answer, target field will be None
     # first element is a list with fields to remove, and attribute that
     # must be None. a dict is a raw answer to pass into `build` method
-    OPTIONAL_FIELDS: tuple[list[tuple[str, str]], dict[str, Any]] | None = None  # noqa: ANN401
+    OPTIONAL_FIELDS: tuple[list[tuple[str, str]], dict[str, Any]] | None = None
```

Comment:
Hm, apparently not, `flake8-annotations` doesn't seem to be capable of detecting `Any` in nested  types, so say `list[Any]` would not be a violation of ANN401. The docs mention this: https://github.com/sco1/flake8-annotations#dynamic-typing-caveats

Additionally, variables (including class variables) aren't enforced to not contain `Any` at all, `flake8-annotations` only checks `Any` in functions (returns or arguments).

Ruff replicates this behavior and also doesn't consider `Any` in nested types to be an issue.

---

However I have also noticed that we actually don't seem to even have `ANN401` enabled. This is an optionally enforced rule, and needs to be selected (`flake8 --extend-select=ANN401`), but if you take a look at the `.flake8` config file (now removed) didn't contain `extend-select=ANN401`, so apparently we weren't even enforcing this in the first place.

I suppose the noqas were added manually, because of the use of Any was suspected to be an issue, without actually checking? I'm not quite sure. However I did put together this minimal example, to show how `flake8-annotations` behaves on various things, and indeed it only reports issues when `Any` is used directly, and only as a function return/argument annotation:

<details>
<summary>Example code</summary>

```python
from typing import Any, ClassVar, TypeVar

T = TypeVar("T")

a: Any = 5
b: list[Any] = [1, "a", 3.5]


class Foo:
    X: ClassVar[Any] = 10
    Y: Any = 50


def func1(x: list[Any]) -> object:
    ...


def func2(x: Any) -> object:
    ...


def func3(x: object) -> Any:
    ...


def func4(x: object) -> list[Any]:
    ...
```

With just flake8 and flake8-annotations installed, running `flake8 --extend-select=ANN401 --show-source main.py` shows this output:

```
main.py:18:11: ANN401 Dynamically typed expressions (typing.Any) are disallowed
def func2(x: Any) -> object:
          ^
main.py:22:28: ANN401 Dynamically typed expressions (typing.Any) are disallowed
def func3(x: object) -> Any:
                           ^
```

</details>